base-notebook/Dockerfile

```diff
@@ -131,8 +131,8 @@ RUN wget --quiet "https://github.com/conda-forge/miniforge/releases/download/${m
 # Do all this in a single RUN command to avoid duplicating all of the
 # files across image layers when the permissions change
 RUN conda install --quiet --yes \
-    'notebook=6.1.5' \
-    'jupyterhub=1.2.2' \
+    'notebook=6.1.6' \
```

Comment:
I've been following the convo about updating to JupyterLab 3.0 and waiting for extension compatibility. I'll add that the newly preferred backend for JupyterLab 3.0 is `jupyter_server` instead of `notebook`: https://jupyterlab.readthedocs.io/en/stable/getting_started/changelog.html#jupyter-server. This has implications for extensions that have both server and frontend components, as I found out when updating https://github.com/parente/jupyterlab-quickopen for 3.0 (e.g., maintain compatibility with both backends? move to jupyter_server alone? how to support binders which deploy notebook by default and not jupyter_server yet?) Not all of these issues are relevant to docker-stacks, but may suggest we need to make a decision about removing notebook from these images down the line, and pointing people to `nbclassic` and the new *Simple* toggle in JupyterLab for a classic-like experience.

Reply:
I agree is not so simple I will add this information to the dedicated issue #1207. We could work on it together to define a strategy. We have to be careful since images are used in different context (binder, k8s, standalone, community, etc.).